Modules/System/Email/src/Message/EmailMessage.Codeunit.al

```diff
@@ -115,6 +151,26 @@ codeunit 8904 "Email Message"
         Recipients := EmailMessageImpl.GetRecipients(RecipientType);
     end;
 
+    /// <summary>
+    /// Sets the recipents of a certain type of the email message.
+    /// </summary>
+    /// <param name="RecipientType">Specifies the type of the recipients.</param>
+    /// <param name="Recipients">Specifies the list of the recipients' email addresses as a semicolon (;) separated list.</param>
+    procedure SetRecipients(RecipientType: Enum "Email Recipient Type"; var Recipients: Text)
```

Comment:
There's no need for the `var` here.

Reply:
You are correct, it was late when I copied those :-) I resolved them